normalize_repos/labels.json

```diff
@@ -0,0 +1,163 @@
+{
+  "groups": [
+    {
+      "name": "priority",
+      "labels": [
+        {
+          "name": "critical",
+          "color": "UNFAVOURABLE",
+          "description": "Issue breaks production and needs fixing asap",
+          "emoji": "\uD83D\uDFE5"
+        },
+        {
+          "name": "high",
+          "color": "NEGATIVE",
+          "description": "Issue stalls works on the project or its dependents",
+          "emoji": "\uD83D\uDFE7"
+        },
+        {
+          "name": "medium",
+          "color": "NEUTRAL",
+          "description": "Issue is not up actively harmful but that can escalate",
```

Comment:
I'm not sure what this description means. Maybe something like "Issue is not blocking but should be fixed soon."?

Reply:
Issues can be for new features, which doesn't have anything to do with harming users by their absence.

Reply:
Updated in caf892e.